app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/GameGenieUseCase.kt

```diff
@@ -86,6 +107,28 @@ class GameGenieUseCase @Inject constructor(
     }
   }.flowOn(Dispatchers.IO)
 
+  fun loadCompanionChat(packageName: String): Flow<GameGenieChat?> = flow {
```

Comment:
couldnt all this be done with to maps or flatmaps on the original flow ? instead of a new method, one to get the companionchat and another to map it